src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,54 @@
+package bit.javaoop;
+
+import java.awt.*;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class Manager extends AbstractEmployee {
```

Comment:
Jeden rodzaj Managera. Miało być dwóch. Wymusiłoby to wtedy implementacje abstrakcyjnej klasy `AbstractManager` (a przynajmniej takie mieliśmy założenie).